iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/ScalingParameters.java

```diff
@@ -80,4 +91,20 @@ public ScalingParameters setIterative(boolean iterative) {
         this.iterative = iterative;
         return this;
     }
+
+    public static ScalingParameters load() {
+        return load(PlatformConfig.defaultConfig());
+    }
+
+    public static ScalingParameters load(PlatformConfig platformConfig) {
+        Objects.requireNonNull(platformConfig);
+        ScalingParameters scalingParameters = new ScalingParameters();
+        platformConfig.getOptionalModuleConfig("scaling-default-parameters").ifPresent(config -> {
+            scalingParameters.setScalingConvention(config.getEnumProperty("scalingConvention", Scalable.ScalingConvention.class, DEFAULT_SCALING_CONVENTION));
+            scalingParameters.setConstantPowerFactor(config.getBooleanProperty("constantPowerFactor", DEFAULT_CONSTANT_POWER_FACTOR));
```

Comment:
To centralize the default parameter allocation you could also do something like:
```java
optModuleConfig.flatMap(c -> c.getOptionalBooleanProperty("xxx")).orElse(DEFAULT_VALUE);
```

Reply:
The default boolean values might not be identical (it is coincidental that they are all `false` here), it might change in the future (or we might add other boolean attributes with a `true` default value) which is why I prefer to have one default value by attribute.